scripts/deblur

```diff
@@ -391,21 +422,23 @@ def build_biom_table(seqs_fp, output_biom_fp, min_reads, file_type, log_level, l
 @click.option('--overwrite', '-w', required=False, type=bool, default=False,
               is_flag=True,
               show_default=True, help="Overwrite output directory if exists.")
-@click.option('--read-error', '-e', required=False, type=float, default=0.05,
+@click.option('--read-error', required=False, type=float, default=None,
               show_default=True,
               help="Maximal bound on 1 nucleotide read error rate."
-                   "Required if --eror-dist is not supplied")
+                   "If supplied, overrides the --eror-dist!")
 @click.option('--mean-error', '-m', required=False, type=float, default=0.005,
               show_default=True,
               help="The mean per nucleotide error, used for original "
               "sequence estimate. If "
               "not passed typical illumina error rate is used")
-@click.option('--error-dist', '-d', required=False, type=str, default=None,
-              show_default=True,
+@click.option('--error-dist', '-d', required=False, type=str,
+              default=None,
+              show_default=False,
               help="A comma separated list of error probabilities for each "
                    "hamming distance. The length of the list determines the "
                    "number of hamming distances taken into account. "
-                   "This replaces of the --read-error value")
+                   "Ignored if --read-error is supplied.\n"
```

Comment:
This is inaccurate: it is not ignored, an exception is raised. Suggest simply stating that this option is exclusive w.r.t. `--read-error`